<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGISRedCreateProjectDialog</class>
 <widget class="QDialog" name="QGISRedCreateProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QGISRed: Create Project</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/plugins/QGISRed/images/iconLayers.png</normaloff>:/plugins/QGISRed/images/iconLayers.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Project's name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tbNetworkName">
         <property name="text">
          <string>Network</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Project directory:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tbProjectDirectory">
         <property name="text">
          <string>Temporal folder</string>
         </property>
         <property name="cursorPosition">
          <number>15</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="cursorMoveStyle">
          <enum>Qt::VisualMoveStyle</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btSelectDirectory">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>CRS:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tbCRS">
         <property name="text">
          <string>WGS 84</string>
         </property>
         <property name="cursorPosition">
          <number>6</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="cursorMoveStyle">
          <enum>Qt::VisualMoveStyle</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btSelectCRS">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Units:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbUnits">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>LPS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>LPM</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MLD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CMH</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CMD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CFS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GPM</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MGD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>IMGD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AFD</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Headloss Formula:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbHeadloss">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>D-W</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>H-W</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C-M</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btCreateProject">
         <property name="text">
          <string>Create Project</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
